packages/transaction-controller/src/helpers/PendingTransactionTracker.ts

```diff
@@ -353,6 +379,12 @@ export class PendingTransactionTracker {
     return blocksSinceFirstRetry >= requiredBlocksSinceFirstRetry;
   }
 
+  #cleanTransactionToForcePoll(txMeta: TransactionMeta) {
```

Comment:
Minor, should we pass just the ID instead as that's all we need?

Reply:
Good point. Done.